### PR TITLE
Update singleton_thread_locals.py

### PR DESCRIPTION
**Pull request by RobinsonMa**
fix

### CODE REVIEW

File: examples/providers/singleton_thread_locals.py [hunk 1]
```diff
@@ -15,7 +15,7 @@ def example(example_object, queue):
 thread_local_object = providers.ThreadLocalSingleton(object)
 
 # Create singleton provider for thread-safe queue:
-queue = providers.Singleton(Queue.Queue)
+queue = providers.ThreadSafeSingleton(queue.Queue)
```
Comment (rmk135) [inline examples/providers/singleton_thread_locals.py:18]: Please, rename `queue` to `queue_factory`. As of now, `queue` variable overrides the name of the imported module. It's a bad practice. Also `queue_factory` name will be consistent with `thread_factory` at line 26.

**Comment by rmk135** [comment]
@RobinsonMa , thank you for contribution. It will be on production today.

**Comment by RobinsonMa** [comment]
@rmk135 ,It's my pleasure

**Comment by rmk135** [comment]
@RobinsonMa , the code is on production http://python-dependency-injector.ets-labs.org/providers/singleton.html#singleton-providers-and-multi-threading

I've added you to CONTRIBUTORS file. It's distributed with an every copy - https://github.com/ets-labs/python-dependency-injector/blob/master/CONTRIBUTORS.rst

Changelog is also updated. See version `3.16.1` at http://python-dependency-injector.ets-labs.org/main/changelog.html

Roman

**Pull request merged**